ExtensionHandler/Linux/src/AzureRM.py

```diff
@@ -24,12 +28,32 @@
 import urllib.request, urllib.parse, urllib.error
 import shlex
 
+from aria import LogManager, EventProperties, PiiKind, LogConfiguration, LogManagerConfiguration
+
 MAX_RETRIES = 3
 
 configured_agent_exists = False
 agent_configuration_required = True
 root_dir = ''
 handler_utility = None
+event_logger = None
+
+class EventLogger:
+  tenant_token = "5413d41fa2bc4f969c283b8b79f23488-c3090565-2202-41ce-bb2c-76f26d7575ee-7318"
+
+  def __init__(self):
+    log_config = LogConfiguration(log_level=logging.DEBUG)
+    configuration = LogManagerConfiguration(log_configuration = log_config)
```

Comment:
Just a code style consistency nitpick.
Line 45: `(log_level=logging.DEBUG)` - no leading or trailing space around `=`
Line 46: `(log_configuration = log_config)` - leading and trailing space around `=`

Reply:
It will get better once we merge your PRs and add `black` into the repo.

Reply:
Thanks for pointing this out. Will take care.